server/tests/models/categories.spec.js

```diff
@@ -0,0 +1,28 @@
+import chai, { assert, expect } from "chai";
+import chaiAsPromised from "chai-as-promised";
+import models from "../../models/index";
+import { category } from "../fixtures/models/categories";
+
+const { Categories, sequelize } = models;
+
+chai.use(chaiAsPromised);
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true });
+});
+
+describe("Categories table model ", () => {
+  it("should create a Category table with valid category", async () => {
+    const articleCategory = await Categories.create(category);
+    assert.instanceOf(articleCategory, Categories);
+    assert.lengthOf(Object.keys(articleCategory.dataValues), "4");
+  });
```

Comment:
Add `whitespace`here for readability.
```
 it("should create a Category table with valid category", async () => {
    const articleCategory = await Categories.create(category);
    assert.instanceOf(articleCategory, Categories);
    assert.lengthOf(Object.keys(articleCategory.dataValues), "4");
  });

  it("should delete a Category table", async () => {
    const createdCategory = await Categories.create(category);
    assert.equal(createdCategory.dataValues.category, category.category);
  });
```
##### instead of 
```
 it("should create a Category table with valid category", async () => {
    const articleCategory = await Categories.create(category);
    assert.instanceOf(articleCategory, Categories);
    assert.lengthOf(Object.keys(articleCategory.dataValues), "4");
  });
  it("should delete a Category table", async () => {
    const createdCategory = await Categories.create(category);
    assert.equal(createdCategory.dataValues.category, category.category);
  });
```